forc-plugins/forc-doc/src/cli.rs

```diff
@@ -24,6 +24,7 @@ pub struct Command {
     /// file in the current directory or any parent directory.
     #[clap(long)]
     pub manifest_path: Option<String>,
+    pub path: Option<String>,
```

Comment:
We can probably use, @sdankel's suggestion here:

```suggestion
    #[clap(long, alias="manifest-path")]
    pub path: Option<String>,
```